[2023-01-13T08:48:06.887+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_avaible scheduled__2023-01-11T00:00:00+00:00 [queued]>
[2023-01-13T08:48:06.944+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_avaible scheduled__2023-01-11T00:00:00+00:00 [queued]>
[2023-01-13T08:48:06.945+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-01-13T08:48:06.945+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-01-13T08:48:06.945+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-01-13T08:48:07.030+0000] {taskinstance.py:1383} INFO - Executing <Task(HttpSensor): is_api_avaible> on 2023-01-11 00:00:00+00:00
[2023-01-13T08:48:07.040+0000] {standard_task_runner.py:55} INFO - Started process 140 to run task
[2023-01-13T08:48:07.046+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'is_api_avaible', 'scheduled__2023-01-11T00:00:00+00:00', '--job-id', '91', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmp4qt0xa30']
[2023-01-13T08:48:07.049+0000] {standard_task_runner.py:83} INFO - Job 91: Subtask is_api_avaible
[2023-01-13T08:48:07.231+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.is_api_avaible scheduled__2023-01-11T00:00:00+00:00 [running]> on host 274e1da73cdf
[2023-01-13T08:48:07.422+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=is_api_avaible
AIRFLOW_CTX_EXECUTION_DATE=2023-01-11T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-01-11T00:00:00+00:00
[2023-01-13T08:48:07.424+0000] {http.py:120} INFO - Poking: api/
[2023-01-13T08:48:07.448+0000] {base.py:71} INFO - Using connection ID 'user_api' for task execution.
[2023-01-13T08:48:07.450+0000] {http.py:148} INFO - Sending 'GET' to url: https://randomuser.me/api/
[2023-01-13T08:48:07.931+0000] {base.py:213} INFO - Success criteria met. Exiting.
[2023-01-13T08:48:07.950+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=is_api_avaible, execution_date=20230111T000000, start_date=20230113T084806, end_date=20230113T084807
[2023-01-13T08:48:08.042+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-01-13T08:48:08.148+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
